shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -479,10 +482,21 @@ void updateSemantics(ByteBuffer buffer, String[] strings) {
 
         Set<SemanticsObject> visitedObjects = new HashSet<SemanticsObject>();
         SemanticsObject rootObject = getRootObject();
+        SemanticsObject newRoute = null;
         if (rootObject != null) {
           final float[] identity = new float[16];
           Matrix.setIdentityM(identity, 0);
           rootObject.updateRecursively(identity, visitedObjects, false);
+          newRoute = rootObject.getMostSpecificRoute();
+        }
+
+        // Dispatch a TYPE_WINDOW_STATE_CHANGED event if the most recent route changed from the previous
+        // route name.
```

Comment:
Rephrase this comment since it is no longer based on names?

Reply:
Done